release/jobs/health_monitor/templates/health_monitor.yml.erb

```diff
@@ -141,6 +141,12 @@ plugins:
       <% end %>
   <% end %>
 
+   <% if p("hm.syslog_event_forwarder_enabled") %>
```

Comment:
This is literally taken from how all the other plugins are enabled - which seems to work. If we change it for this plugin we should probably do it for all of them in an additional change?